Automattic/wp-calypso
# My Home: Blogger checklist

#### Changes proposed in this Pull Request

* Align blogger tasks with EuqfSnWpfYx8fgiBlVmbuA-fi-4019%3A1522
* Handle the action of each task

![image](https://user-images.githubusercontent.com/13596067/145573101-9a30cee3-8e43-4650-83b1-4b7f2a5a79ce.png)

![image](https://user-images.githubusercontent.com/13596067/145573077-788eb5e1-1217-4ef0-b313-633bbe37f0a2.png)

![image](https://user-images.githubusercontent.com/13596067/145573144-8872a209-36af-41e9-9f86-67ba350f22f0.png)

![image](https://user-images.githubusercontent.com/13596067/145573165-f11047b3-83c1-4162-87c2-fdf885162a80.png)

![image](https://user-images.githubusercontent.com/13596067/145573191-7a877a2d-eb34-4608-9ecf-2cb15e506d1c.png)

![image](https://user-images.githubusercontent.com/13596067/145573216-2d0362c7-14ce-4530-975e-4b92f77dec37.png)

![image](https://user-images.githubusercontent.com/13596067/145573245-663feff6-1b58-4e35-aa3c-744d3f6bf65e.png)

#### Testing instructions



* Apply D71330-code to your sandbox and ensure your API points to the sandbox
* Go to My Home with write intent site
* Check the tasks is correct or not



Related to https://github.com/Automattic/wp-calypso/issues/58372
